modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -336,7 +338,22 @@ var CBIWifiFrequencyValue = form.Value.extend({
 					'VHT20', '20 MHz', htmodelist.VHT20,
 					'VHT40', '40 MHz', htmodelist.VHT40,
 					'VHT80', '80 MHz', htmodelist.VHT80,
+					'VHT80+80', '160 MHz (dual 80 MHz)', htmodelist['VHT80+80'],
 					'VHT160', '160 MHz', htmodelist.VHT160
+				],
+				// 'ad': [
+				// 	'VHT20', '20 MHz', htmodelist.VHT20,
+				//	'VHT40', '40 MHz', htmodelist.VHT40,
+				//	'VHT80', '80 MHz', htmodelist.VHT80,
+				//	'VHT80+80', '160 MHz (dual 80 MHz)', htmodelist['VHT80+80'],
+				//	'VHT160', '160 MHz', htmodelist.VHT160
```

Comment:
- AD uses VHT [source](https://www.rohde-schwarz.com/uk/applications/802.11ad-wlan-at-60-ghz-white-paper_230854-50625.html)

Reply:
At least QCA/Wilocity wil6210 reports 11n-like rates afaik, @lynxis and @robimarko will know more about that.

Reply:
Unfortunately, I don't have the wil6210 based Mikrotik now.
If I remember I can check later what it returns.

Reply:
just to clarify, this is about the phyrate from iwinfo?

Reply:
@lynxis It's about `htmode` as used in `/etc/config/wireless` and reported by `iwinfo`. Afaik only `HT20` is allowed on wil6210 and that translates into **2160 MHz** channel width, right?

Reply:
source: https://github.com/openwrt/openwrt/pull/2417#issuecomment-617744558